### DISCUSSION TITLE
Use packaged

### DISCUSSION THREAD

**Discussion by xcgydfjjjderg** [pyg-team/pytorch_geometric#5865] [Q&A]

**Comment by EdisonLeeeee** [comment]
What is your input `adj`? Does it a `torch.sparse.SparseTensor`?

**Reply by EdisonLeeeee** [reply]
Yeah, a native pytorch sparse tensor is not supported in PyG. You need to convert it to a `torch_sparse.SparseTensor` instead:
```python
from torch_sparse import SparseTensor
adj_t = SparseTensor.from_torch_sparse_coo_tensor(adj).t()
```

**Reply by rusty1s** [reply]
We will be using https://github.com/pyg-team/pytorch_geometric/issues/5867 to track further progress.
